### PR TITLE
Fix `wasm_export_functions` when using a return statement

### PR DESCRIPTION
**Pull request by bkchr**
When used a return statement before we directly returned from the function which is obviously wrong.

### CODE REVIEW

File: client/executor/src/integration_tests/mod.rs [hunk 1]
```diff
@@ -918,3 +918,14 @@ fn unreachable_intrinsic(wasm_method: WasmExecutionMethod) {
...
 	}
 }
+
+test_wasm_execution!(return_value);
+fn return_value(wasm_method: WasmExecutionMethod) {
+	let mut ext = TestExternalities::default();
+	let mut ext = ext.ext();
+
+	assert_eq!(
+		call_in_wasm("test_return_value", &[], wasm_method, &mut ext).unwrap(),
```
Comment (pepyakin) [inline client/executor/src/integration_tests/mod.rs:928]: where is the declaration of that method?

make sure to leave a comment there that the 'return' statement is crucial for that test

**Review by pepyakin — APPROVED** [review]

File: client/executor/src/integration_tests/mod.rs (hunk 1 above)
Comment (pepyakin) [reply client/executor/src/integration_tests/mod.rs:928]: found it, it was hidden because of the whitespaces

Comment (bkchr) [reply client/executor/src/integration_tests/mod.rs:928]: https://github.com/paritytech/substrate/pull/12107/files#diff-4cb81001042364564417da256407594e0a778f68fe68a29531931df0951e00b9R356  already there :)

**Review by cheme — APPROVED** [review]
(I did read it this morning :)

**Pull request merged**